superset/dao/base.py

```diff
@@ -76,6 +76,19 @@ def find_by_ids(cls, model_ids: List[int]) -> List[Model]:
             ).apply(query, None)
         return query.all()
 
+    @classmethod
+    def find_all(cls) -> List[Model]:
```

Comment:
I know this is very similar to `find_by_ids` but decided to go explicit over dry. We're already pretty meta here so didn't want to add a layer to safe a few lines of code.